tasks/native.py

```diff
@@ -0,0 +1,53 @@
+from invoke import task
+from tasks.util import COVID_DIR, BUILD_DIR, DATA_DIR
+from os import makedirs
+from os.path import exists, join
+from shutil import rmtree
+from subprocess import run
+
+
+@task(default=True)
+def build(ctx, clean=False):
+    """
+    Build the native binary
+    """
+    if clean and exists(BUILD_DIR):
+        rmtree(BUILD_DIR)
+
+    makedirs(BUILD_DIR, exist_ok=True)
+
+    run(
+        "cmake -G Ninja {}".format(COVID_DIR),
+        shell=True,
+        check=True,
+        cwd=BUILD_DIR,
+    )
+
+    run("cmake --build . --target all", shell=True, check=True, cwd=BUILD_DIR)
+
+
+@task
+def unzip(ctx):
```

Comment:
This was done in the Dockerfile before so made it hard to do outside the container